docs/rocketchat/get-help-in-rocketchat.md

```diff
@@ -1,95 +1,99 @@
+     - A Mac, a Windows machine, an iPhone, an Android, or another device?
 
-Also explain what device you were using when you experienced the problem. Were you using a Windows machine, a Mac, an iPhone, an Android or something else? If you were using a web application, what browser were you using? Provide as much detail as possible, including version numbers.
+ ## The 'when'
 
----
-## What are you trying to do?
-
- Explain what task you were trying to accomplish.
- 
--  How you were doing it, and why it was important. The more detail you can provide the better. 
-- Let the community know what you were trying to do and what outcome you had expected.
-
----
-## What is the problem?
+ Timestamps help! Please consider letting us know: 
+- The date
+- The time, including time zone if different from P.S.T. 
+- If working on a legacy task, the timestamp of the original work
 
-Now that the community knows what was supposed to happen, tell them what went wrong. Include descriptions of
 
-- How the problem occurred
-- What the problem looks like and how it’s behaving. For example, if a screen is failing to load properly, explain how it looks and why it’s not correct. 
-- Provide as much detail as possible. If you received an error message, include a screenshot of the message with your question. 
+## Post the question
 
-##  What have you done to fix the problem?
+When you finish writing the detailed question, please revisit [Jon Skeet's checklist](https://codeblog.jonskeet.uk/2012/11/24/stack-overflow-question-checklist/) to make sure you're being efficient. Then, post it in the best Rocket.Chat channel. 
 
-The results of your initial troubleshooting efforts can be very useful to the community when they’re trying to determine the cause of your issue. Make sure you include any steps you’ve already taken to resolve the issue, and include as much detail as possible. 
+If you're not being proactive about your steps, then the community might miss the question in favour of responding to the ones specific for the channel. 
 
-If you’ve been following instructions on a StackOverflow thread, provide a link to the thread. If you’ve been reading documentation about the problem, provide a link to the documentation. 
+If unsure of where to post or other questions and/or concerns, connect with us at developer.experience@gov.bc.ca.
 
-## What kind of help do you want?
-What are you asking the community for? Do you want a solution to the problem? A link to documentation you can use? An alternative way to complete a task? Let the community know what kind of support you’re looking for.
-
-## Posting a question
-Send your message in the appropriate Rocket.Chat channel and check back often to see if you’ve gotten a reply. Read the [Rocket.Chat etiquette page](rocketchat-etiquette.md) for more information on how to use Rocket.Chat, and what to do if you don’t get a satisfactory answer.
-
-
----
 ---
```

Comment:
Not sure we want to retain "Related pages" section. It is an artifact from Platform Services original docs.

Reply:
+1 on this, but curious to talk big picture strategy

Reply:
@jenreiher-bcgov @sheaphillips 

Can we remove this section altogether for now, since there's a chance IM/IT of the guide might change? It was part of the original piece.

Reply:
Section removed for now, I'll see impacts and keep in mind for next iteration.